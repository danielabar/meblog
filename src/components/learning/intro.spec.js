import React from "react"
import { render } from "@testing-library/react"
import "@testing-library/jest-dom"

import Intro from "./intro"

describe("Learning Intro", () => {
  it("renders correctly", () => {
    const container = render(<Intro />)
    expect(container).toMatchSnapshot()
  })
})
