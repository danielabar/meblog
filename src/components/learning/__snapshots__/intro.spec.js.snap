// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Learning Intro renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        data-testid="learning-intro"
      >
        Welcome to the Learning section. In a world where change is the only constant, it's crucial to keep learning and expanding our knowledge. This is especially true in software development, where new languages, tools, libraries, and frameworks are always emerging. Here, I've gathered a collection of online courses that I've taken and compiled detailed notes on, so you can learn from my experiences and take your skills to the next level. Also check out my article on how to
         
        <a
          class="markdown-link"
          href="https://danielabaron.me/blog/learning-from-screencasts/"
        >
          get the most out of online courses
        </a>
        , and start your learning journey today!
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      data-testid="learning-intro"
    >
      Welcome to the Learning section. In a world where change is the only constant, it's crucial to keep learning and expanding our knowledge. This is especially true in software development, where new languages, tools, libraries, and frameworks are always emerging. Here, I've gathered a collection of online courses that I've taken and compiled detailed notes on, so you can learn from my experiences and take your skills to the next level. Also check out my article on how to
       
      <a
        class="markdown-link"
        href="https://danielabaron.me/blog/learning-from-screencasts/"
      >
        get the most out of online courses
      </a>
      , and start your learning journey today!
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
